### ISSUE TITLE
Map navigation controls not visible in windows high contrast mode

### ISSUE THREAD

**Issue by PascalvanOol** [mapbox/mapbox-gl-js#5784]
**mapbox-gl-js version**: latest mapbox sample page

### Steps to Trigger Behavior

1. Change your windows theme to high contrast #1
2. Open IE 11 browser
3. Open "Display map navigation controls" example:
https://www.mapbox.com/mapbox-gl-js/example/navigation/

### Expected Behavior
Navigation controls to show a icon in contrasting color.

### Actual Behavior
Navigation controls are shown all black.

**Comment by jfirebaugh** [comment]
From my research, to fix this we need to avoid using `background-image`, which is ignored by IE11 in high-contrast modes. Instead, we should be using inline SVG with a rule such as:

```
@media screen and (-ms-high-contrast: active) {
  .button-icon {
    fill: ButtonText;
  }
}
```

https://css-tricks.com/accessible-svgs-high-contrast-mode/

**Comment by kkaefer** [comment]
Fixed by #8874. Please see https://github.com/mapbox/mapbox-gl-js/pull/8874#issuecomment-548376940 for screenshots of how it looks in Windows High Contrast mode.
